server/item/category/category.go

```diff
@@ -0,0 +1,61 @@
+package category
```

Comment:
Would move this file to `item/category.go` and get rid of the category package.

Reply:
What are your thoughts on instead moving it to the `creative` package? The category is only utilized in creative, anyways.

Reply:
That sounds fair but I think that caused import cycles too.

Reply:
Both alternatives seem to cause import cycles